### PR TITLE
2505 allow retrospectively link a case to an event on android app

### PR DESCRIPTION
**Pull request by sergiupacurariu**

### CODE REVIEW

File: sormas-app/app/src/main/java/de/symeda/sormas/app/caze/read/CaseReadEventListFragment.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	protected void onLayoutBinding(FragmentFormListLayoutBinding contentBinding) {
+		LinearLayoutManager linearLayoutManager = new LinearLayoutManager(getActivity(), RecyclerView.VERTICAL, false);
+		contentBinding.recyclerViewForList.setLayoutManager(linearLayoutManager);
+		contentBinding.recyclerViewForList.setAdapter(eventListAdapter);
+	}
+
+	@Override
+	protected String getSubHeadingTitle() {
+		Resources r = getResources();
+		return "Case events";
```
Comment (BarnaBartha) [inline sormas-app/app/src/main/java/de/symeda/sormas/app/caze/read/CaseReadEventListFragment.java:71]: same here, please use a resource not hard coded string

File: sormas-app/app/src/main/java/de/symeda/sormas/app/caze/edit/CaseEditActivity.java [hunk 2]
```diff
@@ -295,6 +307,41 @@ public void goToNewView() {
...
+				}
+
+			} else {
+				// create event and then link it to the case
+				EventNewActivity.startActivityFromCase(getContext(), getRootUuid());
+			}
+		});
+	}
+
+	private boolean eventLinkedToCase(Case caze, Event event) {
```
[the remark this answers is not in the document]
Comment (sergiupacurariu) [reply sormas-app/app/src/main/java/de/symeda/sormas/app/caze/edit/CaseEditActivity.java:336]: changed

File: sormas-app/app/src/main/java/de/symeda/sormas/app/caze/edit/CaseEditEventListFragment.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+	@Override
+	public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+		adapter.setOnListItemClickListener(this);
+		return super.onCreateView(inflater, container, savedInstanceState);
+	}
+
+	@Override
+	protected String getSubHeadingTitle() {
+		Resources r = getResources();
+		return "Case events";
```
[the remark this answers is not in the document]
Comment (sergiupacurariu) [reply sormas-app/app/src/main/java/de/symeda/sormas/app/caze/edit/CaseEditEventListFragment.java:59]: changed

File: sormas-app/app/src/main/java/de/symeda/sormas/app/caze/read/CaseReadEventListFragment.java (hunk 1 above)
Comment (sergiupacurariu) [reply sormas-app/app/src/main/java/de/symeda/sormas/app/caze/read/CaseReadEventListFragment.java:71]: changed

**Pull request merged**